addons/website/static/tests/tours/conditional_visibility.js

```diff
@@ -74,3 +94,57 @@ tour.register('conditional_visibility_2', {
     },
 },
 ]);
+
+tour.register("conditional_visibility_3", {
+    test: true,
+    url: "/?enable_editor=1",
```

Comment:
I want to discourage this way of having a test entering in edit mode. See [here](https://github.com/odoo/odoo/pull/118849#issuecomment-1523042583).